test/e2e/cmd/root-test

```diff
@@ -87,4 +119,15 @@ $start_cmd &
 ROOT_PID=$!
 test_root_delay $ROOT_PID $DELAY_IN_SEC
 
+if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+  mock_trigger_time=$(date --date "${DELAY_IN_SEC} seconds" +%Y-%m-%dT%T%:z)
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+  mock_trigger_time=$(date -v+${DELAY_IN_SEC}S +%Y-%m-%dT%T%z | sed 's@^.\{22\}@&:@')
+fi
+echo "foo: $mock_trigger_time"
```

Comment:
thanks for adding this check! Could you update the 'foo' to mock-trigger-time or remove the echo completely?

Reply:
seems it was too late already when i wrote this 😄 
removed it now